python/ngraph_bridge/__init__.in.py

```diff
@@ -188,10 +189,15 @@ def is_logging_placement():
 def is_grappler_enabled():
     return ngraph_bridge_lib.ngraph_tf_is_grappler_enabled()
 
+def are_variables_enabled():
```

Comment:
Also add a function to return the CXX11ABI (i.e., a python function that returns an int reflecting the ngraph_bridge_lib.ngraph_tf_cxx11_abi_flag)
Something like: 
```
int cxx11_abi_flag() { return ngraph_bridge_lib.ngraph_tf_cxx11_abi_flag }
```

Reply:
It's already there, no?
https://github.com/NervanaSystems/ngraph-tf/blob/master/python/ngraph_bridge/__init__.in.py#L195

Reply:
oh, python function is missing. Working on it.